### ISSUE TITLE
malware

### ISSUE THREAD

**Issue by Le0Developer** [af8ert/game#1]

**Comment by sacp** [comment]
That is what I was suspecting.

**Comment by sauuuceee** [comment]
has anyone installed it yet?

**Comment by zkingboos** [comment]
Yes, it's. Take a look into scan result of virustotal.

![result](https://media.discordapp.net/attachments/444960864714358784/954753175024455771/unknown.png?width=451&height=141)

**Comment by DeathTBO** [comment]
Yes, malware for sure. Was just given this link and a phony discord nitro link at the same time.

**Comment by drillskibo** [comment]
#4 guys, let's be efficient on this one.

**Comment by Le0Developer** [comment]
> #4 guys, let's be efficient on this one.

Reported it before opening the issue and github doesn't delete repositories on weekends.
Since it's Monday now, they'll delete it soon, so reporting it 100 times is useless.

Last time they deleted it at ~9am UTC, so they're a bit slow this time.
